README.md

```diff
@@ -23,6 +23,11 @@ access to all of its features. Your target system must have a functional [SDL2 l
 * macOS: `brew install sdl2`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev`
 
+Install [RISCOF](https://riscof.readthedocs.io/en/stable/installation.html#install-riscof).
```

Comment:
I think the url should use the home page of riscof. Like the following:
```
[RISCOF](https://riscof.readthedocs.io/en/stable/)
```

Reply:
Got it.

Reply:
`riscof` is only required if you want to execute riscv-arch-tests. So, move to the dedicated section.

Reply:
https://github.com/sysprog21/rv32emu/blob/b70dcaa86dac3adc76b24a939fb7414906e2447e/README.md?plain=1#L71-L74

What about the old description of running risv-arch-test?

Reply:
Exactly. Improve this description.

Reply:
Is there a way to run the old framework now?

Reply:
No, old 2.x framework is deprecated. We shall no longer rely on it.